scss/_buttons.scss

```diff
@@ -43,38 +57,82 @@
   &.disabled,
   fieldset:disabled & { // stylelint-disable-line selector-no-qualifying-type
     pointer-events: none;
+    // Remove CSS gradients if they're enabled
+    background-image: if($enable-gradients, none, null);
     opacity: $btn-disabled-opacity;
     @include box-shadow(none);
   }
 }
 
+.btn-outline {
+  color: var(--btn-accent-color);
+  background-color: transparent;
+  border-color: currentColor;
+
+  &:hover {
+    background-color: var(--btn-accent-color);
```

Comment:
If we put here `--btn-accent-color` for hover `background-color` and for `color` library user can't make these values different without styles overwrite or setting own selectors with variables values.

For example I want to set `color` as `red`, but `background-color` as `green`
I will need to write:
```
.btn-outline {
  --btn-accent-color: red;
  &:hover {
    --btn-accent-color: green;
  }
}
```
I think better do config just with set of variables without writing new nested selectors. What do you think?
And, btw, isn't there color will be the same as background color or not?

Reply:
If we use approach similar to https://github.com/twbs/bootstrap/pull/31708 with placing in SCSS styles only SCSS variables, but in SCSS variables set CSS variables, we could do:

```
// Styles
.btn-outline {
  color: $btn-color;
  &:hover {
     background-color: $btn-hover-bg-color;
  }
}

// Default variables, will do the same as in PR
$btn-color: var(--btn-accent-color) !default;
$btn-hover-bg-color: var(--btn-accent-color) !default;

// In application can changed to e.g.:
$btn-color: var(--btn-color);
$btn-hover-bg-color: var(--btn-hover-bg-color);

// In application custom styles - setting variant without knowing selectors structure
.btn-custom {
  --btn-color: red;
  --btn-hover-color-bg: green;
}